package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "leaderboard-project",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
+    "server": "node server.js",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Medini-Rajendra/leaderboard-project.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Medini-Rajendra/leaderboard-project/issues"
+  },
+  "homepage": "https://github.com/Medini-Rajendra/leaderboard-project#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.5.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.25.3",
+    "hint": "^6.1.9",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-cli": "^4.9.1",
+    "webpack-dev-server": "^4.5.0"
+  },
+  "dependencies": {
+    "loadash": "^1.0.0",    
+    "express": "^4.17.1",
+    "webpack": "^5.64.1",
+    "webpack-dev-middleware": "^5.2.2",
```

Comment:
- [ ] good job setting up webpack 💪 however all webpack dependencies should be under `devDependencies`.

Reply:
I have added webpack dependencies to devDependencies as per ESLint suggestion. Here is the ESLint error, 

2:17  error    'webpack' should be listed in the project's dependencies, not devDependencies                 import/no-extraneous-dependencies
3:30  error    'webpack-dev-middleware' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies

That's why I added to devDependencies